evalai/utils/config.py

```diff
@@ -6,10 +6,14 @@
 
 AUTH_TOKEN_FILE_NAME = 'token.json'
 
+HOST_URL_FILE_NAME = 'host_url'
+
 AUTH_TOKEN_DIR = "{}/.evalai/".format(expanduser('~'))
```

Comment:
I don't know how I missed this earlier, but this should be `expanduser("~/.evalai/")`. There is no need for `.format()` since there are no arguments per se.